packages/knip/src/plugins/relay/index.ts

```diff
@@ -0,0 +1,60 @@
+import parseArgs from 'minimist';
+import type { IsPluginEnabled, Plugin, ResolveEntryPaths } from '../../types/config.js';
+import { toDeferResolve, toEntry } from '../../util/input.js';
+import { isInternal, join } from '../../util/path.js';
+import { hasDependency } from '../../util/plugin.js';
+import type { RelayConfig } from './types.js';
+// https://relay.dev/docs/next/guides/compiler/#configuration
+// https://github.com/facebook/relay/blob/main/compiler/crates/relay-compiler/relay-compiler-config-schema.json
+
+const title = 'Relay';
+
+const enablers = ['vite-plugin-relay', '@swc/plugin-relay', 'babel-plugin-relay'];
+
+const isEnabled: IsPluginEnabled = ({ dependencies, config }) =>
+  hasDependency(dependencies, enablers) || 'relay' in config;
```

Comment:
```suggestion
const isEnabled: IsPluginEnabled = ({ dependencies }) => hasDependency(dependencies, enablers);
```

Plugin typically don't need this, and I think I like this consistency (seeing a few instances in other plugins now I'm planning to remove). Users can still enable the plugin using `relay: true`.